src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,63 @@
+package duke;
+
+import java.time.LocalDateTime;
+
+/**
+ * Deadline Class is a type of task a user can create.
+ * A date and time is required as an input as the deadline.
+ */
+public class Deadline extends Task {
+    private LocalDateTime deadline;
+
+    /**
+     * Constructor for Deadline Class
+     * @param task
+     * @param taskType
+     * @param deadline
+     */
+    public Deadline(String task, String taskType, String deadline) {
+        super(task, taskType);
+        this.deadline = LocalDateTime.parse(deadline, formatter);
+    }
+
+    /**
+     * Returns a string representation of the Deadline for the user to view.
+     * @return String representation of the Deadline.
+     */
+    public String toString() {
+        return this.getTaskType() + this.getStatus() + " " + this.getTask() + this.getPeriod();
+    }
+
+    /**
+     * Returns a string representation of date and time of the deadline.
+     * @return String representation of date and time of the deadline.
+     */
+    public String getPeriod() {
+        return " (By: " + dateToString() + ")";
+    }
+
+    /**
+     * Informs the user a new Deadline has been created.
+     * @return String informing a new Deadline has been created.
+     */
+    public String announcement() {
+        return "New Deadline created!";
+    }
+
+    /**
+     * Returns a string representation of the Deadline for txt file saving purposes.
+     * @return String representation of the Deadline for txt file saving purposes.
+     */
+    public String saveString() {

```

Comment:
Is there an easier way other than using multiple concatenation operators? Would the following work for you? Actually, I think this is personally preference, but would like to let you know of alternatives. Concatenation could be more readable too, since variables and string can be read sequentially.

<img width="465" alt="Screenshot 2024-02-07 at 7 07 14 PM" src="https://github.com/nus-cs2103-AY2324S2/ip/assets/66112309/6a786c30-1f83-46e4-8922-c79a991de553">

I noticed similar instances elsewhere.